nuxt/pages/posts/_name/stakeholders.vue

```diff
@@ -72,3 +72,16 @@ export default {
   }
 }
 </script>
+
+<style scoped>
+.row {
+  margin-right: auto;
```

Comment:
Pas de css sur les class Bootrstrap. Tu peux ajouter la class `mr-auto` sur la row à la place https://getbootstrap.com/docs/4.0/utilities/flex/#auto-margins

Reply:
@KevinBerthier j'ai modifié ce que tu as suggérer, can u please check if it's all good ? thnak u